Meadowlark-js/backends/meadowlark-mongodb-backend/src/repository/Db.ts

```diff
@@ -42,6 +44,12 @@ export async function getNewClient(): Promise<MongoClient> {
       .collection(AUTHORIZATION_COLLECTION_NAME);
     await authorizationCollection.createIndex({ clientName: 1 });
 
+    // Create concurrency collection if not exists
+    const concurrencyCollection: Collection<ConcurrencyDocument> = newClient
+      .db(databaseName)
+      .collection(CONCURRENCY_COLLECTION_NAME);
+    await concurrencyCollection.createIndex({ meadowlarkId: 1, documentUuid: 1 }, { unique: true });
```

Comment:
I think these need to be split into separate indexes, since sometimes you don't have the meadowlarkId. I also wondering if they need to be unique or not. I don't think so, so think about that.

Reply:
Okay, I'll add or modify the tests to see what I can conclude about this. Thanks.

Reply:
Based on the tests I ran, I don't think we need to split the indexes. So, I am thinking we can keep it as it is.